src/async_impl/client.rs

```diff
@@ -94,7 +90,6 @@ impl ClientBuilder {
     /// This is the same as `Client::builder()`.
     pub fn new() -> ClientBuilder {
         let mut headers: HeaderMap<HeaderValue> = HeaderMap::with_capacity(2);
-        headers.insert(USER_AGENT, HeaderValue::from_static(DEFAULT_USER_AGENT));
```

Comment:
Instead of just removing this, can the logic be moved into its own `default_useragent` method (similarly to what we do for `default_headers`)?